File: src/main/java/org/jukeboxmc/event/inventory/InventoryEvent.java
package org.jukeboxmc.event.inventory;

import org.jukeboxmc.event.Event;
import org.jukeboxmc.inventory.Inventory;

/**
 * @author LucGamesYT
 * @version 1.0
 */
public class InventoryEvent extends Event {

    private Inventory inventory;

    public InventoryEvent( Inventory inventory ) {
        this.inventory = inventory;
    }

    public Inventory getInventory() {
        return this.inventory;
    }
}

File: JukeboxMC-API/src/main/kotlin/org/jukeboxmc/api/event/inventory/InventoryEvent.kt
package org.jukeboxmc.api.event.inventory

import org.jukeboxmc.api.event.Event
import org.jukeboxmc.api.inventory.Inventory

open class InventoryEvent(private val inventory: Inventory) : Event() {

    fun getInventory(): Inventory {
        return this.inventory
    }
}